action.yml

```diff
@@ -0,0 +1,64 @@
+name: "Deep Security Smart Check"
+description: "Scan container images with Deep Security Smart Check."
+inputs:
+  DSSC_IMAGE_NAME:
+    description: "Container repository, eg myorg/myimage."
+    required: true
+  DSSC_SMARTCHECK_HOST:
+    description: "Deep Security Smart Check url, eg mydomain.com."
+    required: true
+  DSSC_SMARTCHECK_USER:
+    description: "Deep Security Smart Check username, eg admin."
+    required: true
+  DSSC_SMARTCHECK_PASSWORD:
+    description: (MANDATORY) Deep Security Smart Check password, eg 12345.
```

Comment:
If we are prefixing this description with  `(MANDATORY)` , we should do it to the ones above too. If there's no functionality or markup associated with this though, we should delete it. `MANDATORY` is implied by `required: true`

Reply:
Very good point, I missed that... Better follow the ```required: true``` in all of the inputs that are MANDATORY